lib/converters/index.js

```diff
@@ -40,7 +40,7 @@ function convert(subtitleText, inputExtension, outputExtension, options) {
   if (validationError) throw Error(validationError);
 
   // read inputFile, convert to standardized JSON format
-  const { data, status: parseStatus } = parse(subtitleText, inputExtension, options);
+  const { data, status: parseStatus } = parse(subtitleText, getExtension(subtitleText), options);
```

Comment:
If `getExtension` returns `undefined` will the error throw `File type undefined is not supported...` and is there a way to be clearer with this error message?

Reply:
For documentation: as you mentioned in the comment below, an invalid input file will not make it to this point, the error will be thrown in the joi-validation.